.github/workflows/python-app.yml

```diff
@@ -5,11 +5,10 @@ name: Python application
 
 on:
   push:
-    branches: [ master ]
+    branches: [ CCHQ_Smoke_Test ]
```

Comment:
Why are these branches changing? This would mean that the workflow would only run when code was pushed / PR'd to this branch.